MedicalDocManagment.WebUI/app/users-list/page/item.component.ts

```diff
@@ -0,0 +1,24 @@
+﻿import { Component, Input } from '@angular/core';
+import { Response } from '@angular/http';
+
+import { ItemActionListComponent } from './item-actionlist.component';
+import UserModel from '../../models/usermodel'
+
+@Component({
+  moduleId: module.id,
+  selector: '[item]',
+  templateUrl: './views/item.component.html',
```

Comment:
_If you use `moduleId` you may not use `./` expression to get current file's path, cuz` you have already there ( if you have used `moduleId` property )._ 
It's just information